.coveragerc

```diff
@@ -1590,6 +1590,7 @@ omit =
     homeassistant/components/switchbee/__init__.py
     homeassistant/components/switchbee/const.py
     homeassistant/components/switchbee/switch.py
+    homeassistant/components/switchbee/button.py
```

Comment:
Can you please try to order the files?
We should attempt to keep the list in alphabetical order.